bin/automatic_review.py

```diff
@@ -13,122 +14,145 @@
 review_thresholds = yaml.safe_load(open(cfg_path, 'r'))
 
 
-def main():
```

Comment:
This has been moved to the bottom and merged with `review_runs` and `review_samples`